crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -782,14 +907,25 @@ pub fn queue_prepass_material_meshes<M: Material>(
         .read()
         .get_id::<DrawPrepass<M>>()
         .unwrap();
+    let opaque_draw_deferred = opaque_deferred_draw_functions
```

Comment:
these `DrawDeferred<M>` types are an exact copy of the `DrawPrepass<M>` types. not sure if they need to be different, but i don't think it hurts either... not 100% sure.

Reply:
Good catch! They didn't used to be when I was using the stencil buffer.